omero/sysadmins/version-requirements.txt

```diff
@@ -1750,12 +1743,26 @@ Distribution support
       - N/A
       - N/A
     * - 1.6
-      - N/A
+      - EPEL
       - 21, (22)
       - 14.10, (15.04)
       - (8.0), 7.0 bpo
       - Yes
       - Yes
+    * - 1.8
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+      - Yes
+      - Yes
+    * - 1.9
+      - N/A
+      - N/A
+      - 14.10
```

Comment:
@aleksandra-tarkowska All the versions should be there; this table is firstly a list of the upstream releases of a project, and secondly showing our level of support for each version.  For other components on this page, we have dropped older irrelevant releases, and for this component everything before 1.6 sounds fine.  But having all the later ones there is necessary, and given that the latest Ubuntu LTS is using 1.9, it needs to be there as well.